src/app/domain/osReceive/index.ts

```diff
@@ -1,3 +1,15 @@
 import Minilog from 'cozy-minilog'
 
+import { OsReceiveState } from './models/OsReceiveState'
+
 export const OsReceiveLogger = Minilog('🗃️ OsReceiveService')
+
+export const removeSourceProperty = (
```

Comment:
nit: Maybe we should move this into a logHelper file or similar? (just to give an it that this is not used for business logic)

Reply:
I renamed it to be more clear about its intent. For the file I'm not sure, I think it's a good idea to at least keep it with "osreceivelogger"